package.json

```diff
@@ -1,19 +1,32 @@
 {
   "name": "liquidity-provision-scripts",
-  "version": "0.1.0",
-  "description": "",
+  "description": "Scripts to deploy and manage automated traders on the Gnosis Protocol exchange",
+  "private": true,
```

Comment:
What is this "private" for?

Reply:
It says that the package is not published on NPM. When we finally publish the package we'll remove this line.